src/abacus-ts/calculators/fills.ts

```diff
@@ -0,0 +1,21 @@
+import { IndexerCompositeFillObject } from '@/types/indexer/indexerManual';
+import { keyBy, maxBy, orderBy } from 'lodash';
+
+import { EMPTY_ARR } from '@/constants/objects';
+
+import { MustBigNumber } from '@/lib/numbers';
+
+import { mergeObjects } from '../lib/mergeObjects';
+
+export function calculateFills(
+  liveFills: IndexerCompositeFillObject[] | undefined,
```

Comment:
if we use live prefix in these functions then maybe we should update `ephemeral` -> `live` as well (just for consistencies sake)

Reply:
I like that